modules/cnv-web-console-overview-dashboard.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/architecture
+
+[id="web-console-overview-dashboard_{context}"]
+= Overview Dashboard
+
+* *Details* card provides an brief overview of informational cluster details, including:
+** Cluster name
+** Provider
+** Version number of {CNVProductName} Container Platform.
+* *Inventory* card alerts administrators when direct intervention is required to resolve problems.  Information includes:
```

Comment:
Is the right name for the card? I've only seen it as *Cluster Inventory*. This also applies to *Health*, *Utilization*, and *Capacity*.

Reply:
Thanks! Yes, after reviewing the video you are correct. All changes synced with that view of the UI.